app/scripts/modules/openstack/network/networkSelectField.directive.js

```diff
@@ -26,22 +27,27 @@ module.exports = angular.module('spinnaker.openstack.network.networkSelectField.
         _.defaults(scope, {
           label: 'Network',
           labelColumnSize: 3,
-          floatingIps: []
+          networks: []
```

Comment:
I found a few things that I swear you changed, but yet there they were.  Made me wonder if some of your changes got lost during a rebase/squash.